l10n_it_financial_statements_report/report/templates/financial_statements_report.xml

```diff
@@ -17,10 +17,13 @@
     <template id="report">
         <t t-call="web.html_container">
             <t t-foreach="docs" t-as="o">
-                <t t-call="account_financial_report.internal_layout">
-                    <t
-                        t-call="l10n_it_financial_statements_report.financial_statements_report_base"
-                    />
+                <t t-set="company" t-value="o.company_id" />
+                <t t-call="web.external_layout">
```

Comment:
Ciao, perchè richiami:
 `<t t-call="web.external_layout">` 
in precedenza non era presente, ti serve per qualcosa in particolare?

Reply:
Ciao,
di default in questo report non viene stampato alcun riferimento all'azienda a cui appartiene il bilancio ed in ambito multiazienda non è accettabile.
La maniera più veloce per risolvere questo problema è aggiungere l'header classico `web.external_layout`

Reply:
Anche io sono un po' perplesso da questa modifica:

- Mi sembra strano aggiungere un layout _esterno_ (`web.external_layout`), che quindi è per documenti che vanno distribuiti esternamente, quando il layout precedente era _interno_ (`account_financial_report.internal_layout`); ma credo che questo documento alla fine vada distribuito esternamente quindi magari la nomenclatura era già sbagliata prima.
- Lo stesso problema esiste anche per tutti gli altri report OCA giusto? Allora non dovrebbe essere modificato il layout `account_financial_report.internal_layout`?

Ma queste non sono cose secondo me bloccanti.

Il problema è che applicando due layout viene aggiunto due volte il numero di pagina: nel runboat della `14.0` (che ha queste stesse modifiche da https://github.com/OCA/l10n-italy/pull/3656) ho ottenuto [Financial Statements Report HTML.pdf](https://github.com/OCA/l10n-italy/files/13601166/Financial.Statements.Report.HTML.pdf) che ha 2 pagine: entrambe con il numero di pagina 1/1.
Questo perché il numero di pagina viene aggiunto sia in https://github.com/odoo/odoo/blob/9be75bb17063c6b0207aa82144e9f44b6e49dc45/addons/web/views/report_templates.xml#L472 (è uno dei possibili layout esterni) che in https://github.com/OCA/account-financial-reporting/blob/e8942d398c64827ec1eed883997efbd6196c1c5c/account_financial_report/report/templates/layouts.xml#L21.